libs/sdk-backend-spi/src/workspace/settings/index.ts

```diff
@@ -29,4 +29,22 @@ export interface IWorkspaceSettingsService {
      * @returns promise
      */
     setLocale(locale: string): Promise<void>;
+
+    /**
+     * Sets color palette for current workspace.
+     *
+     * @param colorPaletteId - ID of the color palette to apply to charts in workspace.
+     *
+     * @returns promise
+     */
+    setColorPalette(colorPaletteId: string): Promise<void>;
+
+    /**
+     * Sets theme for current workspace.
+     *
+     * @param themeId - ID of the theme to apply to workspace in organization.
```

Comment:
I think `in organization` might make people think that the function can be used to set a theme ID to all workspaces in an organization, which is not the case, is it?

→ Please update the comment as below instead:

```
@param themeId - ID of the theme to apply to the current workspace.
```